documentation/operators/src/nodes/mix-node-setup.md

```diff
@@ -0,0 +1,700 @@
+
+> Any syntax in `<>` brackets is a user's unique variable. Exchange with a corresponding name without the `<>` brackets.
+
+## Current version
+```
+<!-- cmdrun ../../../../target/release/nym-mixnode --version | grep "Build Version" | cut -b 21-26  -->
+```
+
+The `nym-mixnode` binary is currently one point version ahead of the rest of the platform binaries due to a patch applied between releases.
+
+## Preliminary steps
+
+There are a couple of steps that need completing before starting to set up your mix node:
+
+- preparing your [wallet](https://nymtech.net/docs/wallet/desktop-wallet.html)
+- requisitioning a VPS (Virtual Private Server)
+
+### Wallet preparation
+
+#### Mainnet
+
+Before you initialise and run your mixnode, head to our [website](https://nymtech.net/download/) and download the Nym wallet for your operating system. If pre-compiled binaries for your operating system aren't available, you can build the wallet yourself with instructions [here](https://nymtech.net/docs/wallet/desktop-wallet.html).
+
+If you don't already have one, please create a Nym address using the wallet, and fund it with tokens. The minimum amount required to bond a mixnode is 100 `NYM`, but make sure you have a bit more to account for gas costs.
+
+`NYM` can be purchased via Bity from the wallet itself with BTC or fiat, and is currently present on several [exchanges](https://www.coingecko.com/en/coins/nym#markets).
+
+> Remember that you can **only** use Cosmos `NYM` tokens to bond your mixnode. You **cannot** use ERC20 representations of `NYM` to run a node.
+
+#### Sandbox testnet
+
+Make sure to download a wallet and create an account as outlined above. Then head to our [token faucet](https://faucet.nymtech.net/) and get some tokens to use to bond it.
+
+### VPS Hardware Specs
+
+You will need to rent a VPS to run your mix node on. One key reason for this is that your node **must be able to send TCP data using both IPv4 and IPv6** (as other nodes you talk to may use either protocol).
+
+For the moment, we haven't put a great amount of effort into optimizing concurrency to increase throughput, so don't bother provisioning a beastly server with multiple cores. This will change when we get a chance to start doing performance optimizations in a more serious way. Sphinx packet decryption is CPU-bound, so once we optimize, more fast cores will be better.
+
+For now, see the below rough specs:
+
+- Processors: 2 cores are fine. Get the fastest CPUs you can afford.
+- RAM: Memory requirements are very low - typically a mix node may use only a few hundred MB of RAM.
+- Disks: The mixnodes require no disk space beyond a few bytes for the configuration files.
+
+## Mix node setup
+
+Now that you have built the codebase, set up your wallet, and have a VPS with the `nym-mixnode` binary, you can set up your mix node with the instructions below.
```

Comment:
Missing a link or note to build the repo first here - remember we also have the binaries in the github releases page